src/index.html

```diff
@@ -11,7 +11,156 @@
+        </label>
+
+        <label class="group">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="group">
+          How old are You?
+          <input
+            type="number"
+            name="how old are you"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="group">
+          Full date of birth:
+          <input
+            type="date"
+            name="full_date_of_birth"
+          >
+        </label>
+
+        <label class="group">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="i accept"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="margin-buttom">
+        <legend>Registration</legend>
+        <label class="group">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="group">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="5"
+            maxlength="25"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="margin-buttom">
+        <legend>An interesting fact about you!</legend>
+        <div class="group">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="no"
+            >
+              No
+          </label>
+        </div>
+
+        <label class="group">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="#000"
+          >
+        </label>
+
+        <label class="group">
+          What time do you go to bed?
+          <input
+            type="date"
+            name="time"
+          >
+        </label>
+
+        <label class="group">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="group">
+          How do you rate our work?
+          <input
+            type="range"
+            value="0"
+            min="0"
+            max="10"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="margin-buttom">
+        <legend>Additional info:</legend>
+        <label class="group">
+          Comments:
+          <textarea name="texterea">
+          </textarea>
```

Comment:
```suggestion
          <textarea name="texterea"></textarea>
```
to prevent this
<img width="316" alt="image" src="https://user-images.githubusercontent.com/53611910/204132974-853fbc39-fe8f-4279-8791-fad8e28fe2dc.png">

Reply:
ок